indico/web/templates/forms/category_picker_widget.html

```diff
@@ -36,10 +37,21 @@
                 $field.val(JSON.stringify(hiddenData));
             }
 
+            $.ajax({
+                url: build_url(Indico.Urls.Categories.info, {category_id: category}),
+                dataType: 'json',
+                error: handleAjaxError,
+                success: function(data) {
+                    if (data) {
```

Comment:
Do you ever expect something falsy here? If you use `jsonify_data()` this cannot happen so no need for the check.